src/DataTypes/DataTypeDateTime64.cpp

```diff
@@ -57,9 +57,20 @@ std::string DataTypeDateTime64::doGetName() const
     return out.str();
 }
 
-void DataTypeDateTime64::serializeText(const IColumn & column, size_t row_num, WriteBuffer & ostr, const FormatSettings & /*settings*/) const
+void DataTypeDateTime64::serializeText(const IColumn & column, size_t row_num, WriteBuffer & ostr, const FormatSettings & settings) const
 {
-    writeDateTimeText(assert_cast<const ColumnType &>(column).getData()[row_num], scale, ostr, time_zone);
+    switch (settings.date_time_output_format)
+    {
+        case FormatSettings::DateTimeOutputFormat::Simple:
+            writeDateTimeText(assert_cast<const ColumnType &>(column).getData()[row_num], scale, ostr, time_zone);
+            return;
+        case FormatSettings::DateTimeOutputFormat::UnixTimestamp:
+            writeDateTimeTextUnixTimestamp(assert_cast<const ColumnType &>(column).getData()[row_num], scale, ostr, time_zone);
```

Comment:
`writeIntText` will output what you need.
No need to implement `writeDateTimeTextUnixTimestamp` function.

Reply:
Fixed. Thanks.